frontend/src/app/core/main-menu/main-menu-toggle.service.ts

```diff
@@ -143,7 +143,8 @@ export class MainMenuToggleService {
   }
 
   public openMenu():void {
-    this.setWidth(this.defaultWidth);
+    const width = parseInt(window.OpenProject.guardedLocalStorage(this.localStorageKey) as string, 10);
+    this.setWidth(width || this.defaultWidth);
     this.changeData.next(this.defaultWidth);
```

Comment:
I may be misunderstanding things, but don't we want to pass the stored width to `this.changeData.next` as well?

```suggestion
    const width = parseInt(window.OpenProject.guardedLocalStorage(this.localStorageKey) as string, 10) || this.defaultWidth);
    this.setWidth(width);
    this.changeData.next(width);
```